org.kde.spectacle.json

```diff
@@ -0,0 +1,75 @@
+{
+  "id": "org.kde.spectacle",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-22.08",
+  "sdk": "org.kde.Sdk",
+  "command": "spectacle",
+  "rename-icon": "spectacle",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--talk-name=org.kde.kglobalaccel",
+    "--talk-name=org.kde.plasmashell"
+  ],
+  "modules": [
+    {
+      "name": "kColorPicker",
```

Comment:
This isn't being built as a shared library, I think it should be with `-DBUILD_SHARED_LIBS=ON`